Mathlib/Data/Fintype/Basic.lean

```diff
@@ -1187,13 +1187,27 @@ end Trunc
 
 namespace Multiset
 
-variable [Fintype α] [DecidableEq α]
+variable [Fintype α] [DecidableEq α] [Fintype β] [DecidableEq β]
 
 @[simp]
 theorem count_univ (a : α) : count a Finset.univ.val = 1 :=
   count_eq_one_of_mem Finset.univ.nodup (Finset.mem_univ _)
 #align multiset.count_univ Multiset.count_univ
 
+/-- If f is a bijection of finite sets, it maps universes into universes. -/
+@[simp]
+theorem map_univ_eq_univ_of_bijective (f : α → β) (hf : Function.Bijective f) :
+    map f (Finset.univ : Finset α).val = univ.val := by
```

Comment:
```suggestion
/-- If `e` is an equivalence of finite types, then `Multiset.map e` sends the universal finite set
to the universal finite set. -/
@[simp]
theorem _root_.Equiv.multiset_map_univ_eq_univ (e : α ≃ β) :
    map e univ.val = univ.val := by
```

Reply:
Looks like there's actually a Finset version called `Finset.map_univ_equiv` so I'll name this analogously. LMK if you really want this name.